pkg/storaged/content-views.jsx

```diff
@@ -891,11 +891,15 @@ export class VGroup extends React.Component {
                     SelectOne("purpose", _("Purpose"),
                               {
                                   value: "block",
-                                  choices: purposes
+                                  choices: purposes,
+                                  validate: val => {
+                                      if (val == "vdo" && need_vdo_install && client.features.packagekit_details.reason == "immutable-os")
+                                          return cockpit.format(_("The $0 package needs to be added to the OS image."), vdo_package);
```

Comment:
These 2 added lines are not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18240-20230125-114047-de98707b-fedora-37-devel//Coverage/lcov/github-pr.diff.gcov.html)